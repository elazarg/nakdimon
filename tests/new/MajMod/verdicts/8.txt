הנתבעות מִתְנַגְדוֹת לַבַּקָּשָׁה. לטענתן, ב"כ התובעות מֵעוֹלָם לֹא כַּוָּנָה "שקרן" והפרוטוקול יִרְאֶה שהתייחסותו ותגובתו "הַקָּשָׁה" שֶׁל פְּרוֹפ' פְרִידְמָן הָיְיתָה להצגת עוֹבְדוֹת ופרשנויות מוטות שֶׁל ב"כ התובעות לִפְסַק הַדִּין שֶׁנִּיתָּן בְּמִסְגֶּרֶת ת"א 08-04-2982 בּוֹ שִׁימֵּשׁ פְּרוֹפ' פְרִידְמָן כמומחה מִטַּעַם הנתבעים. פְּרוֹפ' פְרִידְמָן הֵעִיד כִּי הוּא מַפְנֶה מְטוּפָּלִים לְבִיצּוּעַ בְּדִיקוֹת גֵּנֵטִיּוֹת גַּם בְּבֵית הַחוֹלִים הֲדַסָּה, אוּלָם לְטַעֲנַת הנתבעות, אֵין בהפניית חוֹלִים לְבֵית חוֹלִים כְּדֵי לְהַוּוֹת "קִשְׁרֵי עֲבוֹדָה" אֲשֶׁר יֵשׁ בָּהֶם כְּדֵי לְהַשְׁפִּיעַ עַל האובייקטיביות שֶׁל המומחה אוֹ כְּדֵי להטותו כִּבְיָכוֹל לְטוֹבַת הֲדַסָּה. לְגַבֵּי הָעוּבְדָּה שפרופ' פְרִידְמָן מְשַׁמֵּשׁ כרופא יְלָדִים עַצְמָאִי הנותן שֵׁירוּתִים גַּם לשירותי בְּרִיאוּת כְּלָלִית, הֲרֵי שפרופ' פְרִידְמָן נֶחְקַר עַל כָּךְ עַל יְדֵי ב"כ התובעות בֶּעָבָר וְהַדָּבָר אַף עוֹלֶה בְּאוֹפֶן מפורש מֵאֲתַר הָאִינְטֶרְנֶט שֶׁל מָקוֹם עֲבוֹדָתוֹ. מִמֵּילָא אֵין מְדוּבָּר במומחה המועסק עַל יְדֵי כְּלָלִית אוֹ החב כְּלַפֶּיהָ חוֹבוֹת אוֹ הכפוף למרותה, אֶלָּא מְדוּבָּר במומחה בִּתְחוּם הַגֵּנֵטִיקָה אֲשֶׁר בֶּהֱיוֹתוֹ גַּם מוּמְחֶה יְלָדִים מְקַבֵּל בְּאוֹפֶן עַצְמָאִי מְטוּפָּלִים מקופות חוֹלִים בִּתְחוּם מומחיותו הָאַחֵר, וּבֵין הַיֶּתֶר גַּם מְטוּפָּלִים הַחֲבֵרִים בְּקוּפַּת חוֹלִים כְּלָלִית. זֹאת, בְּנוֹסָף לסוגיית העיתוי בּוֹ התבקשה הַבַּקָּשָׁה, לְמַעְלָה מְשַׁנֶּה לְאַחַר הַגָּשַׁת חַוַּות דַּעְתּוֹ שֶׁל פְּרוֹפ' פְרִידְמָן וּלְאַחַר שנשמעו רְאָיוֹת הַתְּבִיעָה בִּמְלוֹאָן ונחקרו מוּמְחֵי הנתבעות.