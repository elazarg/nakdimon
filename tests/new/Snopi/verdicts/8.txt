הַנִּתְבָּעוֹת מִתְנַגְּדוֹת לַבַּקָּשָׁה.
לְטַעֲנָתָן, ב"כ הַתּוֹבְעוֹת מֵעוֹלָם לֹא כּוּנָּה "שֶׁקֶּרֶן" וְהַפְּרוֹטוֹקוֹל יַרְאֶה שֶׁהִתְיַיחֲסוּתוֹ וּתְגוּבָתוֹ "הִקְשָׁה" שֶׁל פרופ' פרידמן הָיְיתָה לְהַצָּגַת עוֹבְדוֹת ופרשנויות מוֹטוֹת שֶׁל ב"כ הַתּוֹבְעוֹת לִפְסַק הַדִּין שֶׁנִּיתֵּן בַּמִּסְגֶּרֶת ת"א 08-04-2982 בּוֹ שִׁימֵּשׁ פרופ' פרידמן כְּמוּמְחֶה מִטַּעַם הַנִּתְבָּעִים.
פרופ' פרידמן הֵעִיד כִּי הוּא מַפְנֶה מְטוּפָּלִים לְבִיצּוּעַ בְּדִיקוֹת גנטיות גַּם בְּבֵית הַחוֹלִים הֲדַסָּה, אוּלָם לְטַעֲנַת הַנִּתְבָּעוֹת, אֵין בְּהַפְנָיַית חוֹלִים לְבֵית חוֹלִים כְּדֵי לְהַוּוֹת "קְשָׁרַי עֲבוֹדָה" אֲשֶׁר יֵשׁ בָּהֶם כְּדֵי לְהַשְׁפִּיעַ עַל הָאוֹבְּיֶיקְטִיבִיּוֹת שֶׁל הַמּוּמְחֶה אוֹ כְּדֵי להטותו כביכול לְטוֹבַת הֲדַסָּה.
לְגַבִּי הָעוּבְדָּה שפרופ' פרידמן מְשַׁמֵּשׁ כְּרוֹפֵא יְלָדִים עַצְמָאִי הַנּוֹתֵן שֵׁירוּתִים גַּם לְשֵׁירוּתֵי בְּרִיאוּת כְּלָלִית, הֲרֵי שפרופ' פרידמן נֶחְקַר עַל כָּךְ עַל יְדֵי ב"כ הַתּוֹבְעוֹת בְּעָבָר וְהַדָּבָר אַף עוֹלֶה בָּאוֹפֶן מפורש מֵאֲתַר הָאִינְטֶרְנֶט שֶׁל מְקוֹם עֲבוֹדָתוֹ.
מִמֵּילָא אֵין מְדוּבָּר בַּמּוּמְחֶה הַמָּועֳסָק עַל יְדֵי כְּלָלִית אוֹ החב כְּלַפֵּיהָ חוֹבוֹת אוֹ הַכָּפוּף למרותה, אֶלָּא מְדוּבָּר בְּמוּמְחֶה בַּתְּחוּם הַגֶּנֶטִיקָה אֲשֶׁר בֶּהֱיוֹתוֹ גַּם מוּמְחֶה יְלָדִים מְקַבֵּל בְּאוֹפֶן עַצְמָאִי מְטוּפָּלִים מִקּוּפּוֹת חוֹלִים בִּתְחוּם מוּמְחִיּוּתוֹ הָאַחֵר, וּבֵין הַיֶּתֶר גַּם מְטוּפָּלִים הַחֲבֵרִים בְּקוּפַּת חוֹלִים כְּלָלִית.
זֹאת, בְּנוֹסָף לסוגיית הָעִיתּוּי בּוֹ הִתְבַּקְּשָׁה הַבַּקָּשָׁה, לְמַעְלָה מִשְׁנָה לְאַחֵר הִגַּשְׁתָּ חַוּוֹת דַּעְתּוֹ שֶׁל פרופ' פרידמן וּלְאַחֵר שֶׁנִּשְׁמְעוּ רְאָיוֹת הַתְּבִיעָה במלואן וְנֶחְקְרוּ מוּמְחֵי הַנִּתְבָּעוֹת.