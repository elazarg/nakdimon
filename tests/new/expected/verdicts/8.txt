הַנִּתְבָּעוֹת מִתְנַגְּדוֹת לַבַּקָּשָׁה.
לְטַעֲנָתָן, ב"כ הַתּוֹבְעוֹת מֵעוֹלָם לֹא כּוּנָּה "שַׁקְרָן" וְהַפְּרוֹטוֹקוֹל יָרְאֶה שֶׁהִתְיַיחֲסוּתוֹ וּתְגוּבָתוֹ "הַקָּשָׁה" שֶׁל פְּרוֹפ' פְרִידְמָן הָיְיתָה לְהַצָּגַת עוּבְדּוֹת וּפַרְשָׁנוּיוֹת מוּטּוֹת שֶׁל ב"כ הַתּוֹבְעוֹת לִפְסַק הַדִּין שֶׁנִּיתַּן בְּמִסְגֶּרֶת ת"א 08-04-2982 בּוֹ שִׁימֵּשׁ פְּרוֹפ' פְרִידְמָן כְּמוּמְחֶה מִטַּעַם הַנִּתְבָּעִים.
פְּרוֹפ' פְרִידְמָן הֵעִיד כִּי הוּא מַפְנֶה מְטוּפָּלִים לְבִיצּוּעַ בְּדִיקוֹת גֵּנֵטִיּוֹת גַּם בְּבֵית הַחוֹלִים הֲדַסָּה, אוּלָם לְטַעֲנַת הַנִּתְבָּעוֹת, אֵין בְּהַפְנָיַית חוֹלִים לְבֵית חוֹלִים כְּדֵי לְהַוּוֹת "קִשְׁרֵי עֲבוֹדָה" אֲשֶׁר יֵשׁ בָּהֶם כְּדֵי לְהַשְׁפִּיעַ עַל הָאוֹבְּיֶיקְטִיבִיּוּת שֶׁל הַמּוּמְחֶה אוֹ כְּדֵי לְהַטּוֹתוֹ כִּבְיָכוֹל לְטוֹבַת הֲדַסָּה.
לְגַבֵּי הָעוּבְדָּה שֶׁפְּרוֹפ' פְרִידְמָן מְשַׁמֵּשׁ כְּרוֹפֵא יְלָדִים עַצְמָאִי הַנּוֹתֵן שֵׁירוּתִים גַּם לְשֵׁירוּתֵי בְּרִיאוּת כְּלָלִית, הֲרֵי שֶׁפְּרוֹפ' פְרִידְמָן נֶחְקַר עַל כָּךְ עַל יְדֵי ב"כ הַתּוֹבְעוֹת בֶּעָבָר וְהַדָּבָר אַף עוֹלֶה בְּאוֹפֶן מְפוֹרָשׁ מֵאֲתַר הָאִינְטֶרְנֶט שֶׁל מְקוֹם עֲבוֹדָתוֹ.
מִמֵּילָא אֵין מְדוּבָּר בְּמוּמְחֵה הַמּוֹעֳסָק עַל יְדֵי כְּלָלִית אוֹ הֶחָב כְּלַפֶּיהָ חוֹבוֹת אוֹ הַכָּפוּף לְמָרוּתָהּ, אֶלָּא מְדוּבָּר בְּמוּמְחֵה בִּתְחוּם הַגֵּנֵטִיקָה אֲשֶׁר בֶּהֱיוֹתוֹ גַּם מוּמְחֵה יְלָדִים מְקַבֵּל בְּאוֹפֶן עַצְמָאִי מְטוּפָּלִים מִקּוּפּוֹת חוֹלִים בִּתְחוּם מוּמְחִיּוּתוֹ הָאַחֵר, וּבֵין הַיֶּתֶר גַּם מְטוּפָּלִים הַחֲבֵרִים בְּקוּפַּת חוֹלִים כְּלָלִית.
זֹאת, בְּנוֹסָף לְסוּגְיַית הָעִיתּוּי בּוֹ הִתְבַּקְּשָׁה הַבַּקָּשָׁה, לְמַעְלָה מִשָּׁנָה לְאַחַר הַגָּשַׁת חַוַּות דַּעְתּוֹ שֶׁל פְּרוֹפ' פְרִידְמָן וּלְאַחַר שֶׁנִּשְׁמְעוּ רְאָיוֹת הַתְּבִיעָה בִּמְלוֹאָן וְנֶחְקְרוּ מוּמְחֵי הַנִּתְבָּעוֹת.
